Description: MFreq Makefile patch for Debian Packaging
 - Change PREFIX in upstream Makefile to '/usr' for package build.
 - Remove line in upstream Makefile stripping executables prior to installation.
Author: Robert James Clay <jame@rocasa.us>
Last-Update: <2015-10-30>

--- a/Makefile
+++ b/Makefile
@@ -14,14 +14,14 @@
 
 # compiler flags
 CC = gcc
-CFLAGS = -fstack-protector -Wall -g -I/usr/include -I. ${DEFINES}
-LDFLAGS =
+CFLAGS += -fstack-protector -Wall -g -I/usr/include -I. ${DEFINES}
+LDFLAGS +=
 
 # libraries to link
 LIBS =
 
 # where to install stuff
-PREFIX = /usr/local
+PREFIX = $$(pwd)/debian/mfreq/usr
 
 # objects
 OBJ_COMMON = log.o misc.o tokenizer.o
@@ -83,7 +83,6 @@
 #
 
 install: ${PROGS}
-	strip ${PROGS}
 	cp -f mfreq-index ${PREFIX}/bin
 	cp -f mfreq-list ${PREFIX}/bin
 	cp -f mfreq-srif ${PREFIX}/bin
